### ISSUE TITLE
car.normal in MultiBUGS

### ISSUE THREAD

**Issue by aspassiani** [MultiBUGS/MultiBUGS#6]
I've appeared to run into a new issue when moving my model from OpenBUGS to MultiBUGS. When I run the model I run into a "trap #100" error that appears to be associated with BlackBox, so this occurs in the part of my script where I set the number of cores. I've determined that if I remove the car.normal part of my model, this issue goes away. I was wondering if it is not possible to use MultiBUGS when the model has a CAR component to it or if there is some kind of modification that must be done to get the model to work with MultiBUGS? 

Thanks for the help,
Alessio

**Comment by rjbgoudie** [comment]
Yes, we are aware that CAR models are not currently working, but this is in development, and will hopefully be available soon!

**Comment by aspassiani** [comment]
Thats great! Being able to run my full model in MultiBUGS would significantly decrease the amount of time it takes to run. Looking forward to the update. 
Thanks!

**Comment by diegopujoni** [comment]
Did you make development with the CAR models? I'm trying to use, but I get this error:

```
Error in bugs.run(n.burnin, MultiBUGS.pgm, debug = debug, WINE = WINE,  : 
  MultiBUGS did not run correctly
Log file:
=================

BugsCmds.SetDisplay('log')

BugsCmds.SetFilePath('C:/Users/diego/Documents/R/win-library/3.5/R2MultiBUGS/model/schools.txt' ); BugsCmds.ParseGuard; BugsCmds.Parse
model is syntactically correct

BugsCmds.SetFilePath( 'C:/Users/diego/AppData/Local/Temp/RtmpIX0fMc/data.txt' ); BugsCmds.LoadDataGuard; BugsCmds.LoadData
data loaded

BugsCmds.CompileGuard; BugsCmds.specificationDialog.numChains  := 3; BugsCmds.Compile
model compiled in 0 s

BugsCmds.SetRNGuard; BugsCmds.SetRNState(1)

BugsCmds.SetFilePath( 'C:/Users/diego/AppData/Local/Temp/RtmpIX0fMc/inits1.txt' ); BugsCmds.LoadInitsGuard; BugsCmds.specificationDialog.chain := 1; BugsCmds.LoadInits
initial values loaded and chain initialized but another chain contains uninitialized variables

BugsCmds.SetFilePath( 'C:/Users/diego/AppData/Local/Temp/RtmpIX0fMc/inits2.txt' ); BugsCmds.LoadInitsGuard; BugsCmds.specificationDialog.chain := 2; BugsCmds.LoadInits
initial value
```
Thanks

**Comment by rjbgoudie** [comment]
Hi @diegopujoni, thanks for the report. CAR didn't make the cut for v1.0, but it is mostly implemented in v2.0alpha. A few rough edges still to iron out.

However, did the model you were running actually include CAR? It looks like you are getting an error running the 'schools' model that is included with R2MultiBUGS. This model doesn't involve CAR, unless you've edited it. The default schools model runs fine for me with v1.0 and current R2MultiBUGS, but if you are having problems with that, we can look into it.

**Comment by diegopujoni** [comment]
Thank you for your reply, @rjbgoudie ,

The default schools model also ran fine for me using v1.0 and current R2MultiBUGS.
When I changed the model to include the model I'm studying (this model has a CAR structure), so I get the error. How can I download and use the v2.0alpha?
Best,

**Comment by rjbgoudie** [comment]
Closing, since CAR models implemented in v2.0.
